cli/wallet/nep17.go

```diff
@@ -522,33 +522,48 @@ func multiTransferNEP17(ctx *cli.Context) error {
 	if err != nil {
 		return cli.NewExitError(err, 1)
 	}
-	acc, err := getDecryptedAccount(wall, from, pass)
+	acc, err := options.GetUnlockedAccount(wall, from, pass)
 	if err != nil {
 		return cli.NewExitError(err, 1)
 	}
 
 	gctx, cancel := options.GetTimeoutContext(ctx)
 	defer cancel()
 
-	c, err := options.GetRPCClient(gctx, ctx)
-	if err != nil {
-		return cli.NewExitError(err, 1)
-	}
-
 	if ctx.NArg() == 0 {
 		return cli.NewExitError("empty recipients list", 1)
 	}
 	var (
 		recipients      []transferTarget
 		cosignersOffset = ctx.NArg()
 	)
-	cache := make(map[string]*wallet.Token)
+
 	for i := 0; i < ctx.NArg(); i++ {
 		arg := ctx.Args().Get(i)
 		if arg == cmdargs.CosignersSeparator {
 			cosignersOffset = i + 1
 			break
 		}
+	}
+	cosigners, extErr := cmdargs.GetSignersFromContext(ctx, cosignersOffset)
+	if extErr != nil {
+		return extErr
+	}
+	signersAccounts, err := cmdargs.GetSignersAccounts(acc, wall, cosigners, transaction.CalledByEntry)
+	if err != nil {
+		return cli.NewExitError(fmt.Errorf("invalid signers: %w", err), 1)
+	}
+	c, act, err := options.GetRPCWithActor(gctx, ctx, signersAccounts...)
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
+
+	cache := make(map[string]*wallet.Token)
+	for i := 0; i < ctx.NArg(); i++ {
+		arg := ctx.Args().Get(i)
+		if arg == cmdargs.CosignersSeparator {
+			break
+		}
```

Comment:
nope

Reply:
OK, that's because `cosignersOffset` has two separate meanings. Apply the following changes in a separate commit:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ git diff
diff --git a/cli/wallet/nep17.go b/cli/wallet/nep17.go
index 9fbd72f0..23e6fb14 100644
--- a/cli/wallet/nep17.go
+++ b/cli/wallet/nep17.go
@@ -535,16 +535,16 @@ func multiTransferNEP17(ctx *cli.Context) error {
        }
        var (
                recipients      []transferTarget
-               cosignersOffset = ctx.NArg()
+               cosignersOffset = ctx.NArg() // `--` position.
        )
        for i := 0; i < ctx.NArg(); i++ {
                arg := ctx.Args().Get(i)
                if arg == cmdargs.CosignersSeparator {
-                       cosignersOffset = i + 1
+                       cosignersOffset = i
                        break
                }
        }
-       cosigners, extErr := cmdargs.GetSignersFromContext(ctx, cosignersOffset)
+       cosigners, extErr := cmdargs.GetSignersFromContext(ctx, cosignersOffset+1)
        if extErr != nil {
                return extErr
        }
```
And after that you can safely remove the `if arg == cmdargs.CosignersSeparator ` check.